dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -341,6 +341,23 @@ TaskFunction createFramePolicyIntegrationTest() {
   };
 }
 
+Map<String, dynamic> _average(List<Map<String, dynamic>> results, int iterations) {
+  final Map<String, dynamic> tally = <String, dynamic>{};
+  for (Map<String, dynamic> item in results) {
+    item.forEach((String key, dynamic value) {
+      if (tally.containsKey(key)) {
+        tally[key] = (tally[key] as int) + (value as int);
+      } else {
+        tally[key] = value;
+      }
+    });
+  }
+  tally.forEach((String key, dynamic value) {
+    tally[key] = (value as int) ~/ iterations;
```

Comment:
Why not making `tallky[key]` as `double` to get more precision?

Reply:
I'm keeping the same datatypes as before to avoid breakage.  With how big these numbers are 1/3 of a number is inconsequential.